emperor/support_files/js/controller.js

```diff
@@ -505,7 +505,7 @@ define([
 
     // this is the number of expected view controllers that will announce that
     // all view controllers have been successfully loaded.
-    this._expected = 5;
+    this._expected = 7;
```

Comment:
Should this be self calculated? Seems hard to have to chance as controllers are added ...